src/viur/core/skeleton.py

```diff
@@ -1968,34 +1968,60 @@ def canCall(self) -> bool:
         return user is not None and "root" in user["access"]
 
     def dataSkel(self):
-        modules = ["*"] + listKnownSkeletons()
-        modules.sort()
         skel = BaseSkeleton().clone()
-        skel.module = SelectBone(descr="Module", values={x: translate(x) for x in modules}, required=True)
+
+        skel.skelname = SelectBone(
```

Comment:
It's the kindname, not the skelname. By skelname I understand the name of the class, i.e. FileLeafSkel, while the child name is file. It is also consistent with the attribute in the `Skeleton` / `SkeletonInstance`.

```suggestion
        skel.kindname = SelectBone(
```